can/interfaces/vector/canlib.py

```diff
@@ -73,20 +67,20 @@ def __init__(
         channel: Union[int, Sequence[int], str],
         can_filters: Optional[CanFilters] = None,
         poll_interval: float = 0.01,
-        receive_own_messages: bool = False,
-        bitrate: int = None,
+        receive_own_messages: Optional[bool] = False,
+        bitrate: Optional[int] = None,
         rx_queue_size: int = 2 ** 14,
-        app_name: str = "CANalyzer",
-        serial: int = None,
-        fd: bool = False,
-        data_bitrate: int = None,
+        app_name: Optional[str] = "CANalyzer",
+        serial: Optional[int] = None,
+        fd: Optional[bool] = False,
```

Comment:
Why should we allow `None`? It either is FD-enabled or not.

Reply:
Fixed.